enterprise/backend/test/metabase_enterprise/transforms/api_test.clj

```diff
@@ -226,6 +226,26 @@
                   list-resp (mt/user-http-request :crowberto :get 200 "ee/transform")]
               (is (seq list-resp)))))))))
 
+(deftest list-transforms-returns-transforms-in-id-order-test
```

Comment:
For next time I don't think you really need an extra test for something like this. In the past when I've fixed flakes like this I've just added the `:order-by` and called it a day... I don't think this test really does anything to prove that you're doing an explicit `:order-by` any more than the original flaky test did, since when it's indeterminate it seems to come back in ID order most of the time anyway. Like if you removed that `:order-by` then this test and the original flaky test would probably still pass the majority of the time.

Reply:
Yeah that's fair. Will drop the test.